UI/Web/src/app/user-settings/manage-reading-profiles/manage-reading-profiles.component.html

```diff
@@ -10,13 +10,13 @@
     </div>
 
     <p class="ps-2">{{t('description')}}</p>
-    <p class="ps-2 text-muted">{{t('extra-tip')}}</p>
+    <p class="ps-2 text-muted">{{t('extra-tip')}} <a target="_blank" [href]="WikiLink.ReadingProfiles">{{t('wiki-title')}}</a></p>
```

Comment:
Anchors leaving the instance need to have rel= set on them to avoid leaking any user domain information.